pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/housekeeping/handler/EmailNotificationHandler.java

```diff
@@ -89,6 +99,32 @@ public void handleMessage(NotificationMessage message) {
             }
         }
 
+        //skip emails for PlayWright generated users / Email deny list
+        if (!emailDenyPatterns.isEmpty()) {
+            boolean hasDenyListEmail = false;
+            List<String> nonDenyEmailList = new ArrayList<>();
+            for (String toAddress : message.getDistributionList()) {
```

Comment:
If I'm reading this correctly,  if all addresses on the distribution list match the deny pattern, the email is not sent. How many addresses typically appear on the distribution list? Besides the ptp, what other addresses does the list contain?
A comment might help explain some of this.

Reply:
Usualy only ptp email address, however code/platform does support more  than 1 to email hence needed to handle it.